File: Company/src/main/webapp/service/nalogZaPrenosService.js
/**
 * Created by Olivera on 22.6.2017..
 */

app.service('nalogZaPrenosService', function($http){
    return {
        read: function(onSuccess, onError){
            $http.get('/api/naloziZaPrenos').then(onSuccess, onError);
        },
        readDuznik: function(naziv, onSuccess, onError){
            $http.get('/api/naloziZaPrenos/firmaDuznik/' + naziv + '/').then(onSuccess, onError);
        },
        kreirajNalog: function (podaciZaNalog, onSuccess, onError) {
            $http.post('api/naloziZaPrenos/kreirajNalog', podaciZaNalog).then(onSuccess, onError);
        }
    }
});

File: Company/src/main/java/com/ftn/service/NalogZaPrenosService.java
package com.ftn.service;

import com.ftn.model.dto.NalogZaPrenosDTO;
import com.ftn.model.dto.PodaciZaNalogDTO;

import java.util.List;

/**
 * Created by Olivera on 22.6.2017..
 */
public interface NalogZaPrenosService {

    List<NalogZaPrenosDTO> read();

    List<NalogZaPrenosDTO> readDuznik(String naziv);

    NalogZaPrenosDTO create(NalogZaPrenosDTO nalogZaPrenosDTO);

    NalogZaPrenosDTO kreirajNalog(PodaciZaNalogDTO podaciZaNalogDTO);
}
